target/ppc/translate/vmx-impl.c.inc

```diff
@@ -1257,6 +1257,80 @@ static void gen_vsldoi(DisasContext *ctx)
     tcg_temp_free_i32(sh);
 }
 
+static bool trans_VSLDBI(DisasContext *ctx, arg_VN *a)
+{
+    TCGv_i64 t0, t1, t2;
+
+    REQUIRE_INSNS_FLAGS2(ctx, ISA310);
+    REQUIRE_VECTOR(ctx);
+
+    t0 = tcg_temp_new_i64();
+    t1 = tcg_temp_new_i64();
+    t2 = tcg_temp_new_i64();
+
+    get_avr64(t0, a->vra, true);
+    get_avr64(t1, a->vra, false);
+
+    if (a->sh != 0) {
+        /* vrt.h = (vra.h << sh) | (vra.l >> (64 - sh)) */
+        tcg_gen_shli_i64(t0, t0, a->sh);
+        tcg_gen_shri_i64(t2, t1, 64 - a->sh);
+        tcg_gen_or_i64(t0, t0, t2);
+
+        /* vrt.l = (vra.l << sh) | (vrb.h >> (64 - sh)) */
+        get_avr64(t2, a->vrb, true);
+        tcg_gen_shli_i64(t1, t1, a->sh);
+        tcg_gen_shri_i64(t2, t2, 64 - a->sh);
+        tcg_gen_or_i64(t1, t1, t2);
+    }
+
+    set_avr64(a->vrt, t0, true);
+    set_avr64(a->vrt, t1, false);
+
+    tcg_temp_free_i64(t0);
+    tcg_temp_free_i64(t1);
+    tcg_temp_free_i64(t2);
+
+    return true;
+}
+
+static bool trans_VSRDBI(DisasContext *ctx, arg_VN *a)
+{
+    TCGv_i64 t2, t1, t0;
+
+    REQUIRE_INSNS_FLAGS2(ctx, ISA310);
+    REQUIRE_VECTOR(ctx);
+
+    t0 = tcg_temp_new_i64();
+    t1 = tcg_temp_new_i64();
+    t2 = tcg_temp_new_i64();
```

Comment:
Same here

Reply:
Fixed in https://github.com/PPC64/qemu/pull/18/commits/69a1f824a0db9d22f122f26e2ed86da0b34e52ad